pkg/java/jar/parse.go

```diff
@@ -159,11 +159,20 @@ func (p *Parser) parseArtifact(fileName string, size int64, r dio.ReadSeekerAt)
 		return libs, nil, nil
 	}
 
+	partialGAVInfoExist := false
+
 	manifestProps := m.properties()
+
+	if (manifestProps.groupID != "" || manifestProps.artifactID != "") && manifestProps.version != "" {
```

Comment:
Instead of mandating version, should we proceed even if one value is present? i.e ```if len(manifestProps.groupID )>0  || len(manifestProps.artifactID )>0 || len(manifestProps.version )>0```

Reply:
if you do not have manifestProps.groupID and manifestProps.artifactID, but have manifestProps.version then showing only manifestProps.version does not make sense.

Reply:
Sample package object 
```
{      
      "Name": "commons-daemon:commons-daemon",
      "Version": "1.3.1",
      "Layer": {
        "Digest": "sha256:fe7f7dd28c7a625c39a1bbbbc528fd4320dc77877621e3bf2247c3c1e3497fb8",
        "DiffID": "sha256:8c95b20a337217595c495e2bf4690cbb483720a8c2499013c6d27c033f71da5b"
      },
      "FilePath": "usr/local/tomcat/bin/commons-daemon.jar"
}
```
I was thinking from response point of view. We will skip such packages acc to current change. I think it's fine to include them. Infact, if none of the GAV info is present, we should still include the package. Filepath will always be present.

Reply:
ok sure.

Reply:
done pls check

Reply:
Don't see the change. Did you miss committing the change? I still see GAV being validated 
```	if (manifestProps.groupID != "" || manifestProps.artifactID != "") && manifestProps.version != "" {```